src/v/raft/follower_queues.cc

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2020 Vectorized, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#include "raft/follower_queues.h"
+
+#include <seastar/core/coroutine.hh>
+#include <seastar/core/semaphore.hh>
+
+#include <absl/container/node_hash_set.h>
+
+#include <vector>
+
+namespace raft {
+
+follower_queues::follower_queues(
+  vnode self, uint32_t max_concurrent_append_entries)
```

Comment:
should we count bytes or requests or both?

Reply:
Since we are going to implement  proper backpressure propagation over our RPC layer we agreed that it is simpler to track number of requests per follower for now.